### DISCUSSION TITLE
在模组整合包加载时无法正常登陆

### DISCUSSION THREAD

**Discussion by LBY123165** [DreamVoid/MiraiMC#245] [问答 / Q&A] [answered]
服务器是mohist1.12.2版本
起初运行暮色森林，等价交换，ae2等模组可以正常登陆
后期模组换为了rlcraft的模组包无法登陆
输入 mirai login 账号 密码 后，只显示登陆新的机器人账号以及登录协议，此后再无反应
插件包内仅有miraimc，chat2qq，火花及authme登录插件四个，且之前可以使用
请问是与模组有冲突吗

**Comment by LBY123165** [comment]
机器人未被屏蔽，未被tx封禁

**Comment by DreamVoid** [comment, marked as answer]
miraimc在启动阶段就告诉你了不兼容forge
